### DISCUSSION TITLE
Use case being slowly deprecated (Catalog API)

### DISCUSSION THREAD

**Discussion by callumb123** [amzn/selling-partner-api-models#3348] [General]
I am becoming increasingly concerned that my particular use case is being ignored/forgotten about when updates to the catalog API are being introduced. My use case is searching for products with listCatalogItems/searchCatalogItems using product titles/keywords rather than identifiers such as ASIN/UPC/EAN etc.

At the moment I am forced to continue using listCatalogItems instead of searchCatalogItems as in many cases the latter can return drastically different (and worse) results as discussed in #829 - This is despite the restore rate for listCatalogItems being much lower.

It seems that the deprecation of listCatalogItems is going ahead in Sept 2022 despite no improvements being made to searchCatalogItems in this regard. This has potentially devastating consequences to my application and to my clients.

This has been made worse by the recent SP-API Throttling Adjustments, as even if I was able to use searchCatalogItems, since I am using product titles I would be unable to use the batch call for this and therefore my restore rate is being reduced by 60% (5 -> 2).

**Comment by jamesp26** [comment]
The throttling changes are really, really frustrating. It feels to me that tools that clearly screen scrape are being rewarded for their investments in that strategy.  We have invested a huge amount of effort migrating everything over to SPAPI, dealing with the issues along the way, adjusting for the new data formats etc, and to be rewarded with 60% less throughput..

**Comment by callumb123** [comment]
@chapmanjw Can you comment on this as we have discussed this in the past? Every day the deprecation deadline is getting closer and makes me even more concerned about this issue and the future of my business

**Comment by callumb123** [comment]
@chapmanjw Please provide your input on this, we are less than 4 weeks away from the throttle changes that are no good for keyword requests and less than 8 weeks till the deprecation of ListCatalogItems, both of which will have a devastating impact on my business

**Comment by gokigoks** [comment]
Also worried about the throttling in SPAPI. The day of September 30 seems like a nightmare closing in.

**Comment by callumb123** [comment]
@chapmanjw @ShivikaK Only 1 week to go until this deprecation and throttling issue becomes a huge for me and other developers and there has been no comment made from the SP-API development team. Please provide your input here
